container-engine-lib/lib/backend_impls/docker/docker_manager/docker_auth_test.go

```diff
@@ -0,0 +1,97 @@
+package docker_manager
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// WriteStaticConfig writes a static Docker config.json file to a temporary directory
+func WriteStaticConfig(t *testing.T, configContent string) string {
```

Comment:
can we make make this a private helper function if it's not used outside of the package?